resources/public/api/conf/3.0/broughtForwardLosses_list.raml

```diff
@@ -0,0 +1,30 @@
+is:
+  - headers.acceptJson
+  - headers.testScenarioHeader
+  - errors.formatNino
+  - errors.formatTaxYear
+  - errors.ruleTaxYearRangeInvalid
+  - errors.ruleTaxYearNotSupported
+  - errors.notFound
+  - errors.formatBusinessId
+  - errors.formatLossType
+  - errors.clientOrAgentNotAuthorised
+  - queryParameters.businessId
+  - queryParameters.typeOfLossForBFLosses
+  - queryParameters.taxYearForBFLosses
```

Comment:
`taxYearForBFLosses` needs to be removed. You will have to create a new `taxYearBroughtForwardFrom` in the query parameters and call it here instead